SR2019/2018-11-13-rules-announce.md

```diff
@@ -0,0 +1,14 @@
+---
+to: Student Robotics 2019
+subject: Rules for SR2019 published
+---
+
+Hi all,
+
+It's with great pleasure we're publishing the rulebook for Student Robotics 2019. We announced the game at [kickstart](https://www.studentrobotics.org/2018-11-12-sr2019-rules-released/), and gave you an insight into the rules, but the rulebook formalises and documents them for all to see.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the other robots within the zone boundaries.
+
+[Download the rules](https://studentrobotics.org/docs/rules/)
+
+If you have any questions about the rules, please [post on our forums](https://www.studentrobotics.org/forum/).
```

Comment:
Minor: our website redirects to remove `www.`, so we can save visitors a redirect by doing that here too:
```suggestion
If you have any questions about the rules, please [post on our forums](https://studentrobotics.org/forum/).
```